index.html

```diff
@@ -1524,24 +1525,28 @@ <h2>Overview</h2>
                 <span class="rfc2119-assertion" id="arch-td-consumers-process">
                   <a>Consumers</a> MUST be able to process the <a>WoT Thing Description</a> format, which is based on JSON [[!RFC8259]].
                 </span>
-                The format can be processed either through classic JSON libraries or a JSON-LD processor, as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
+                The format can be processed either through classic JSON libraries or a JSON-LD processor, 
+                as the underlying information model is graph-based and its serialization compatible with JSON-LD 1.1 [[?json-ld-syntax]].
                 A <a>TD</a> is instance-specific (i.e., describes an individual Thing, not types of Things)
-                and is the default Web representation of a <a>Thing</a>.
+                and is the default external textual (Web-) representation of a <a>Thing</a>.
```

Comment:
What does external mean?
What is an internal textual representation?
Why a hyphen after Web?

Reply:
An internal representation can be anything - external means that it is available outside of the thing.

Reply:
then
external, textual
(needs comma)